packages/api/server/http.mts

```diff
@@ -128,6 +128,25 @@ router.post('/generate', cors(), async (req, res) => {
   }
 });
 
+// Generate a cell using AI from a query string
+router.options('/sessions/:id/generate_cell', cors());
```

Comment:
Why not a websocket event?

Reply:
I don't have a super strong reason: I favor http when i need request/response semantics with a pending state. It's also less code.

Do you think this would be better as websocket?